addons/common/DisableMouseDialog.hpp

```diff
@@ -0,0 +1,18 @@
+class ctrlMapEmpty;
+class GVAR(DisableMouse_Dialog) {
+    idd = -1;
+    movingEnable = 0;
+    onLoad = QUOTE(uiNamespace setVariable [ARR_2(QUOTE(QGVAR(dlgDisableMouse)),_this select 0)];);
```

Comment:
```suggestion
    onLoad = QUOTE(uiNamespace setVariable [ARR_2(QQGVAR(dlgDisableMouse),_this select 0)];);
```
not tested

Reply:
I really like `with` because you don't have to worry about nested quotes
```suggestion
    onLoad = QUOTE(with uiNameSpace do { GVAR(dlgDisableMouse) = _this select 0; };);
```

Reply:
That's much more readable. Should be improved by using # though.

Reply:
`_this # 0` or ` _this#0`